addons/binding/org.openhab.binding.keba/src/main/java/org/openhab/binding/keba/handler/KeContactBroadcastListener.java

```diff
@@ -0,0 +1,196 @@
+package org.openhab.binding.keba.handler;
```

Comment:
It seems that this file has a missing copyright header, can you fix all of them by running 

`mvn license:format` 

Please commit only the ones for your binding.